### ISSUE TITLE
visualize results

### ISSUE THREAD

**Issue by mbirkhez** [Sara-Ahmed/SiT#3]
Hi, I fine-tuned the model on my custom dataset for object detection and now I want to visualize the images and detected bounding boxes. Any idea how to do that?

**Comment by Sara-Ahmed** [comment]
It depends on how did you train the model for object detection? did you only use the rotation and contrastive tokens? Or did you also use the output tokens corresponding to the image?
What is your loss overall?

**Comment by mbirkhez** [comment]
I did not add any new loss term. I just made my dataset class similar to TinyImageNet's class which is also using the bounding boxes.

**Comment by Sara-Ahmed** [comment]
The main issue is that you wanted to visualise the detected bounding boxes, that means you trained your model to detect bounding boxes rather than classes, is that right?

**Comment by mbirkhez** [comment]
Yes I changed the classification head and the loss for object detection. I figured out the visualization also. Since that is not related to your network so I am closing this issue. Thanks for your great work!
